### PR TITLE
DOCSP-2344: Style and tidy uriwriter

### PR DESCRIPTION
**Pull request by i80and**
Staged at https://docs-mongodbcom-staging.corp.mongodb.com/guides/andrew/guides/guides/server/drivers.html

### CODE REVIEW

File: sphinxext/uriwriter.py [hunk 1]
```diff
@@ -8,63 +8,37 @@
...
+               <li class="uriwriter__toggle guide__pill">on-premise MongoDB with replica set</li>
+               <li class="uriwriter__toggle guide__pill">Atlas (Cloud)</li>
+           </ul>
+       </div>
+       <label class="mongodb-form__prompt uriwriter__atlascontrols">
+           <div class="mongodb-form__label">Atlas connection string</div>
+           <div>
+               <textarea class="mongodb-form__input" id="uriwriter_atlaspaste" spellcheck=false rows="3" cols="50" placeholder='mongo "mongodb+srv://clustername.mongodb.net/test" --username user'></textarea>
+               <div class="atlascontrols__status mongodb-form__status"></div>
+           </div>
```
Comment (skerschb) [inline sphinxext/uriwriter.py:25]: Somewhere in here we need a way to store and display the original string that the user pasted, or some way of letting them know that the string they pasted in a previous visit is x and parsed into the target URISTRINGs

Comment (i80and) [reply sphinxext/uriwriter.py:25]: Done!

File: sphinxext/uriwriter.py [hunk 2]
```diff
@@ -8,63 +8,37 @@
 URIWRITER_TEMPLATE = '''
 .. raw:: html
 
-   <p class="uriwriter">
```
Comment (skerschb) [inline sphinxext/uriwriter.py:11]: node examples are not updating

Comment (i80and) [reply sphinxext/uriwriter.py:11]: Updated the node templates to match the rest

**Pull request merged**